Specification/BrAPI-Schema/BrAPI-Germplasm/Cross.json

```
@@ -65,12 +65,12 @@
                 },
                 "plannedCross": {
                     "$ref": "PlannedCross.json#/$defs/PlannedCross",
-                    "description": "the unique identifier for a planned cross",
+                    "description": "The Planned Cross that was using in the planning of this Cross.",
```

Maybe...

```suggestion
                    "description": "The Cross that was used in the planning of this one.",
```

Feel like this avoids saying the same thing twice, either are probably fine tho, really.